cedar-policy-core/src/extensions/decimal.rs

```diff
@@ -51,6 +51,12 @@ mod names {
         pub static ref GREATER_THAN : Name = Name::parse_unqualified_name("greaterThan").expect("should be a valid identifier");
         pub static ref GREATER_THAN_OR_EQUAL : Name = Name::parse_unqualified_name("greaterThanOrEqual").expect("should be a valid identifier");
     }
+
+    // Global regex, initialized at first use
+    // PANIC SAFETY This is a valid `Regex`
+    lazy_static::lazy_static! {
+        pub static ref DECIMAL_REGEX : Regex = Regex::new(r"^(-?\d+)\.(\d+)$").unwrap();
+    }
 }
```

Comment:
should this be in the same `lazy_static` block as the above?  not sure the tradeoff here